cmd/configsum/config.go

```diff
@@ -154,15 +159,26 @@ func runConfig(args []string, logger log.Logger) error {
 	var (
 		mux          = http.NewServeMux()
 		prefixConfig = fmt.Sprintf(`/%s/config`, apiVersion)
-		clientSvc    = client.NewService(clientRepo, tokenRepo)
+		clientSVC    = client.NewService(clientRepo, tokenRepo)
 		svc          = config.NewServiceUser(baseRepo, userRepo)
```

Comment:
the constructor of client service doesn't follow the user service naming convention. it could be the other way around as well but it should be consistent.